lib/drivers/animasDriver.js

```diff
@@ -1786,8 +1785,23 @@ module.exports = function (config) {
             progress(100);
             return cb(err, data);
           } else {
-            progress(100);
-            return cb(null, data);
+            progress(50);
+
+            cfg.api.upload.toPlatform(
+              data.post_dataservices,
```

Comment:
Sure! As I'm using 'uploads' and 'dataservices' for `uploadType`, I'll use `data.post_uploads` and `data.post_dataservices`.